client/src/API.ts

```diff
@@ -14,13 +14,13 @@ const getTicket = async (service_id:number) => {
   return data;
   }
 
-  const getNextCustomer = async (service_ids: number[]) => {
+  const getNextCustomer = async () => {
     const response = await fetch(`${SERVER_URL}/api/line/next-customer`, {
       method: 'POST',
       headers: {
         'Content-Type': 'application/json'
       },
-      body: JSON.stringify({ service_ids })
+      body: JSON.stringify({  })
```

Comment:
how does it work without service_id?

Reply:
Because, for now, the counter is not linked to services... Each counter do all the services